modules/perception/camera/lib/obstacle/detector/yolo/yolo_obstacle_detector.cc

```diff
@@ -132,11 +145,26 @@ bool YoloObstacleDetector::InitNet(const yolo::YoloParam &yolo_param,
 }
 
 void YoloObstacleDetector::InitYoloBlob(const yolo::NetworkParam &net_param) {
-  auto obj_blob = inference_->get_blob(net_param.obj_blob());
-  int output_height = obj_blob->shape(1);
-  int output_width = obj_blob->shape(2);
+  auto obj_blob_scale1 = inference_->get_blob(net_param.det1_obj_blob());
+  auto obj_blob_scale2 = inference_->get_blob(net_param.det2_obj_blob());
+  auto obj_blob_scale3 = inference_->get_blob(net_param.det3_obj_blob());
+  int output_height_scale1 = obj_blob_scale1->shape(1);
+  int output_width_scale1 = obj_blob_scale1->shape(2);
   int obj_size =
-      output_height * output_width * static_cast<int>(anchors_.size()) / 2;
+    output_height_scale1 * output_width_scale1 *
+    static_cast<int>(anchors_.size()) / 2;
+  if (obj_blob_scale2) {
+    int output_height_scale2 = obj_blob_scale2->shape(1);
+    int output_width_scale2 = obj_blob_scale2->shape(2);
+    int output_height_scale3 = obj_blob_scale3->shape(1);
+    int output_width_scale3 = obj_blob_scale3->shape(2);
+    obj_size =
+      (output_height_scale1 * output_width_scale1 +
+       output_height_scale2 * output_width_scale2 +
+       output_height_scale3 * output_width_scale3) *
+      static_cast<int>(anchors_.size()) / 2 / 3;
```

Comment:
/2/3 to one constrexpr float

Reply:
Here, they are all integers in the expression. So I created 2 `constexpr` for both `2` and `3` and used them here as you suggested above instead of a `constexpr float` for 2/3.